SECURITY_UPDATES_TODO.md

```diff
@@ -0,0 +1,392 @@
+- **Fix**:
+  ```bash
+  cd backend
+  poetry add h11@^0.16.0
+  ```
+
+### 2. python-jose < 3.4.0 (Python/Backend)
+- **Current version**: 3.3.0
+- **Required version**: ≥ 3.4.0
+- **Impact**: JWT signature bypass vulnerability (CVE-2024-33663)
+- **Advisory**: https://nvd.nist.gov/vuln/detail/CVE-2024-33663
+- **Fix**:
+  ```bash
+  cd backend
+  poetry add python-jose@^3.4.0
+  ```
+
+### 3. xmldom ≤ 0.6.0 (NPM/Frontend)
+- **Status**: Transitive dependency (not directly in package.json)
+- **Required version**: > 0.6.0
+- **Impact**: XML parsing vulnerability
+- **Advisory**: https://github.com/xmldom/xmldom/security/advisories/GHSA-crh6-fp67-6883
+- **Fix**: Update parent packages or use npm audit fix
+
+---
+
+## HIGH Severity Vulnerabilities (10)
+
+### Backend (Python)
+
+#### 4. starlette ≤ 0.49.0
+- **Current**: 0.35.1
+- **Required**: > 0.49.0
+- **Impact**: Multiple security issues
+- **Fix**:
+  ```bash
+  cd backend
+  poetry add starlette@^0.49.1
+  ```
+- **Note**: Verify FastAPI compatibility before updating
+
+#### 5-7. urllib3 (Multiple CVEs)
+- **Current**: 1.26.11
+- **Required**: ≥ 2.5.0 (or ≥ 1.26.19 for older branch)
+- **Impact**: Multiple security vulnerabilities
+- **Fix**:
+  ```bash
+  cd backend
+  poetry add urllib3@^2.5.0
+  ```
+- **Warning**: Major version change (1.x → 2.x) - test thoroughly
+
+#### 8. ecdsa ≥ 0
+- **Status**: All versions vulnerable
+- **Required**: Check for updates or alternatives
+- **Impact**: Cryptographic vulnerability
+- **Fix**: May need to wait for upstream fix or use alternative library
+
+### Frontend (NPM)
+
+#### 9-10. playwright < 1.55.1
+- **Current**: 1.42.1
+- **Required**: ≥ 1.55.1
+- **Impact**: Security vulnerabilities in browser automation
+- **Fix**:
+  ```bash
+  cd frontend
+  npm install playwright@^1.55.1
+  ```
+
+---
+
+## MEDIUM Severity Vulnerabilities (24)
+
+### Backend (Python)
+
+#### requests < 2.32.4
+- **Fix**: `poetry add requests@^2.32.4`
+
+#### sentry-sdk < 1.45.1
+- **Fix**: `poetry add sentry-sdk@^1.45.1`
+
+#### certifi < 2024.7.4
+- **Current**: 2021.5.30 to 2024.7.3 range
+- **Fix**: `poetry add certifi@^2024.7.4`
+
+#### aiohttp < 3.12.14
+- **Fix**: `poetry add aiohttp@^3.12.14`
+
+### Frontend (NPM)
+
+#### vite (Multiple CVEs - 8 issues)
+- **Current**: 5.0.13
+- **Required**: ≥ 5.4.20
+- **Impact**: Various XSS and security issues
+- **Fix**:
+  ```bash
+  cd frontend
+  npm install vite@^5.4.20
+  ```
+
+#### @babel/runtime < 7.26.10
+- **Impact**: Denial of Service vulnerability
+- **Fix**: `npm install @babel/runtime@^7.26.10`
+
+#### prismjs < 1.30.0
+- **Status**: Likely transitive dependency
+- **Fix**: `npm install prismjs@^1.30.0` or update parent
+
+---
+
+## LOW Severity Vulnerabilities (6)
+
+These are lower priority but should still be addressed:
+- Various minor versions of vite
+- Other transitive dependencies
+
+---
+
+## Complete Fix Script
```

Comment:
I increasingly feel this file should be an issue and I will give it as much attention as dependabot issue